jaxrs-tck/src/main/java/jakarta/ws/rs/tck/common/webclient/Goldenfile.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2007, 2021 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+/*
+ * $Id$
+ */
+
+package jakarta.ws.rs.tck.common.webclient;
```

Comment:
code common across TCK, can be retained at Jakarta EE TCK repo. we can create common jar for these source, use  common jar across separated TCK with a maven dependency.

Reply:
Agree, could it be better if we do that in separate PRs. When all the tests are migrated we would know the right list of common java files/ jar files from Jakarta EE TCK repo required. Also the task of creating the jars with common source and pushing them to maven is pending.